bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/overheads/Overheads.scala

```diff
@@ -14,8 +14,6 @@ import scala.annotation.tailrec
 sealed trait Overheads {
   // REL-2985 -> REL-2926: acquisitionOverhead and otherOverheadFraction are no longer used.
   def partnerOverheadFraction: Double // fpart
-  def acquisitionOverhead: Time       // acqover
```

Comment:
This is a bit scary but these fields are not being used

Reply:
Nice to see this hot mess being cleaned up.